Makefile

```diff
@@ -34,20 +34,25 @@ dev: build/manifest.json .pydeps
 docker:
 	git archive HEAD | docker build -t hypothesis/hypothesis:$(DOCKER_TAG) -
 
-# Run docker image built with `docker` task
+# Run docker image built with `docker` task.
+# This command is intended only for use in development and assumes that the
+# service containers (Postgres, Elasticsearch etc.) have been run using
+# docker-compose.
 .PHONY: run-docker
 run-docker:
-	$(eval RABBITMQ_CONTAINER ?= rabbitmq)
+	$(eval RABBITMQ_CONTAINER ?= rabbit)
 	$(eval PG_CONTAINER ?= postgres)
+	$(eval ES1_CONTAINER ?= elasticsearchold)
 	$(eval ES_CONTAINER ?= elasticsearch)
 	docker run \
-		--link $(RABBITMQ_CONTAINER) \
-		--link $(PG_CONTAINER) \
-		--link $(ES_CONTAINER) \
+		--net h_default \
 		-e "APP_URL=http://localhost:5000" \
+		-e "AUTHORITY=localhost" \
 		-e "BROKER_URL=amqp://guest:guest@$(RABBITMQ_CONTAINER):5672//" \
 		-e "DATABASE_URL=postgresql://postgres@$(PG_CONTAINER)/postgres" \
-		-e "ELASTICSEARCH_HOST=http://$(ES_CONTAINER):9200" \
+		-e "ELASTICSEARCH_HOST=http://$(ES1_CONTAINER):9200" \
```

Comment:
I believe this is the same docker container as qa/prod in which case it is not using the .ini file with all the dev env vars defined in it so we have to specify the required env vars here before running it. We wouldn't want to put them in the docker-compose because they aren't required for the docker compose when running in make dev. Ideally it would just use the dev .ini file but that may require some changing of the docker container command/config-it's potentially worth taking a quick peak but I'm not going to let it hold up this PR. Please correct me if that doesn't sound correct.

Reply:
Are these url paths actually supposed to work locally cause if so I think these should be set to localhost:9200 etc?

Reply:
The intention is that this command runs h in a "somewhat production-like" environment - debug facilities off, key config settings provided via env vars, running in a Docker container built using `make docker`. That's why it doesn't use the dev ini file.

The primary purpose is to make it easy and quick to locally test changes to the Docker build.